### PR TITLE
updating ocm section to link to existing up to date doc

### PR DESCRIPTION
**Pull request by laurafitzgerald**
# Description
As part of a change triggered by https://issues.redhat.com/browse/INTLY-8098 
we need to add details on supported versions of RHMI and byoc clusters supported versions.
I noticed there was a section in this README and a section here: https://github.com/integr8ly/delorean/tree/master/docs/ocm
Seems the linked one is updated more regularly so pointing to that here and removing the duplicated steps.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ Currently the operator installs the following products:
 - Integreatly solution explorer
 
 ## Prerequisites
+
+NOTE: Due to a change in how networking is configured for openshift in v4.4.6 (mentioned in the [cloud resource operator](https://github.com/integr8ly/cloud-resource-operator#supported-openshift-versions)) there is a limitation on the version of Openshift that RHMI can be installed on for BYOC clusters.
```
Comment (laurafitzgerald) [inline README.md:22]: fyi: This link won't be fully active until this commit is merged. https://github.com/integr8ly/cloud-resource-operator/pull/158/commits/702a2a863ee5fa0ce4bb49d7dd4130319d4ccd56

**Comment by grdryn** [comment]
Hmm, not sure why it didn't take the first time. There was another example of that earlier, so maybe the prow labelling stuff wasn't working for a while or something :thinking:

**Pull request merged**